src/jit/gentree.cpp

```diff
@@ -9193,6 +9193,8 @@ GenTreeUseEdgeIterator::GenTreeUseEdgeIterator(GenTree* node)
 {
     assert(m_node != nullptr);
 
+    // NOTE: the switch statement below must be updated when introducing new nodes.
```

Comment:
Hmm if you're familiar with this code and comment then you know you have to update. Otherwise, well, you'll have no idea that the comment is here. May the mighty assert guide your way toward this switch :).